### PR TITLE
Farm 27 page contactez nous

### PR DESCRIPTION
**Pull request by CaroD59**

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]

File: src/components/Contact/index.jsx [hunk 1]
```diff
@@ -1,11 +1,140 @@
...
+    const newDetails = { ...details };
+    newDetails[evt.target.name] = evt.target.value;
+    setDetails(newDetails);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios.post(`${process.env.REACT_APP_API_URL}/contact`, details).then(
+      (response) => {
+        console.log(response);
```
Comment (LoicBrassart) [inline src/components/Contact/index.jsx]: Nope

File: src/components/Contact/index.jsx [hunk 2]
```diff
@@ -1,11 +1,140 @@
...
+        setIsSent(true);
+      },
+      (error) => {
+        console.log(error);
```
Comment (LoicBrassart) [inline src/components/Contact/index.jsx]: Nope

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "axios": "^0.21.1",
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
+    "nodemailer": "^6.6.2",
```
Comment (LoicBrassart) [inline package.json]: This package doesn't belong here

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
No console logs, and mails are sent from the backeend

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — APPROVED** [review]

File: src/components/Login/index.jsx [hunk 4]
```diff
@@ -51,7 +51,7 @@ export default function LoginForm() {
                   type="password"
                   password="Password"
                   id="Password"
-                  placeholder="Mot de passe"
+                  placeholder="Mot de passe"
```
Comment (LoicBrassart) [inline src/components/Login/index.jsx]: ... Wtf ? o.O

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**